presto-main/src/main/java/com/facebook/presto/operator/scalar/VarbinaryFunctions.java

```diff
@@ -395,7 +395,7 @@ public static long crc32(@SqlType(StandardTypes.VARBINARY) Slice slice)
 
     @Description("compute fnv-1 32 bit")
     @ScalarFunction("fnv1_32")
-    @SqlType(StandardTypes.BIGINT)
+    @SqlType(StandardTypes.INTEGER)
     public static long fnv1Hash32(@SqlType(StandardTypes.VARBINARY) Slice slice)
```

Comment:
I'm not sure if I should change  return type in Java from `long` to `int`, because if I do this, tests fail with:
```
com.facebook.presto.spi.PrestoException: Exact implementation of fnv1_32 do not match expected java types.
```

Reply:
Just checked in the code. Actually all `@SqlType(StandardTypes.INTEGER)` args and return types use `long` as internal Java type. So this code is correct.